prov/efa/src/rxr/rxr_pkt_type_misc.c

```diff
@@ -278,7 +278,6 @@ void rxr_pkt_handle_readrsp_sent(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_en
 
 	tx_entry = (struct rxr_tx_entry *)pkt_entry->x_entry;
 	data_len = rxr_get_readrsp_hdr(pkt_entry->pkt)->seg_size;
-	tx_entry->state = RXR_TX_SENT_READRSP;
```

Comment:
What state is the tx_entry in after the patch? Should we set it to RXR_TX_REQ or RXR_TX_SEND?

Reply:
For eager messages it will be set to `RXR_TX_REQ` in `rxr_tx_entry_init`. For long messages it's set to `RXR_TX_SEND` a few lines later in this function.

Reply:
Thanks!